jenkins/Jenkinsfile.pull-request-build

```diff
@@ -54,6 +69,7 @@ pipeline {
           utils.postReviewCheck(title: "Build Package")
           utils.postReviewCheck(title: "GWT Unit Tests")
           utils.postReviewCheck(title: "C++ Unit Tests")
+          utils.postReviewCheck(title: "Automated Tests")
```

Comment:
This might need to be changed to run only when this same condition is true:

https://github.com/rstudio/rstudio/pull/15475/files#diff-d2886fdd570919cdfe051d263c19071926c4230468179391c56b2d96c8f04e8aR255

Reply:
I'll let you tweak that after it's merged.